### PR TITLE
Remove unused dependencies

### PR DESCRIPTION
**Pull request by figsoda**

### CODE REVIEW

**Comment by edfloreshz** [comment]
Thanks, I'll take a look at it this weekend. 👍🏼

**Comment by edfloreshz** [comment]
The project isn't compiling, do you mind taking a look at it?

Screenshot_20230603_140234

File: Cargo.toml [hunk 1]
```diff
@@ -75,10 +71,6 @@ features = ["macros", "libadwaita", "gnome_43"]
 version = "1.0.126"
 features = ["derive"]
```
Comment (edfloreshz) [inline Cargo.toml:73]: This may fix the issue

```suggestion

[dependencies.gettext-rs]
version = "0.7"
features = ["gettext-system"]
```

Comment (figsoda) [reply Cargo.toml]: thanks, I added it back and it builds now

I think this might be a cargo-machete bug, I will file a bug upstream

**Comment by edfloreshz** [comment]
Thank you! 🎉

**Pull request merged**